### PR TITLE
set default value for time

### PR DESCRIPTION
**Pull request by astridx**
It happens very often that the `<time>`-tag in the gpx file is removed. Currently this plugin throws an error and terminate. My PR would insert a default value if a `<trkseg>`-tag does not contain a `<time>`-tag.

### CODE REVIEW

File: gpx.js [hunk 1]
```diff
@@ -445,6 +445,8 @@ L.GPX = L.FeatureGroup.extend({
       _ = el[i].getElementsByTagName('ele');
       if (_.length > 0) {
         ll.meta.ele = parseFloat(_[0].textContent);
+      } else {
+        ll.meta.time = new Date('1970-01-01T00:00:00');
```
Comment (mpetazzoni) [inline gpx.js]: Are you sure this isn't supposed to be in the block above, line 443?

Comment (astridx) [reply gpx.js]: Sorry. Of course you are right.

File: gpx.js [hunk 2]
```diff
@@ -440,13 +440,15 @@ L.GPX = L.FeatureGroup.extend({
...
       }
 
       _ = el[i].getElementsByTagName('ele');
       if (_.length > 0) {
         ll.meta.ele = parseFloat(_[0].textContent);
       }
-
+
```
Comment (mpetazzoni) [inline gpx.js]: Remove this whitespace (but keep the empty line)

**Comment by mpetazzoni** [comment]
Thanks!

File: gpx.js (hunk 2 above)
Comment (astridx) [reply gpx.js]: I hope it is OK now.

**Pull request merged**